app/Services/PopulateService.php

```diff
@@ -0,0 +1,160 @@
+    {
+        $file = fopen($path . config('scraper.command.buildings_filename'), 'r');
+
+        if ($file) {
+            while (($line = fgets($file)) !== false) {
+                $values = explode(';', $line);
+                $cityName = $values[3];
+                $buildingType = $values[4];
+
+                $cityId = $this->getCityId($cityName, $country);
+                $typeId = $this->getBuildingTypeId($buildingType);
+
+                Building::firstOrCreate([
+                    'name' => $values[0],
+                    'description' => $values[1],
+                    'address' => $values[2],
+                    'type_id' => $typeId,
+                    'city_id' => $cityId,
+                    'main_image_url' => $values[5],
+                    'pending' => false
+                ]);
+            }
+
+            fclose($file);
+        } else {
+            die(config('scraper.command.buildings_filename') . ' file was not found');
+        }
+    }
+    private function populateAmenities(string $path): void
+    {
+        $file = fopen($path . config('scraper.command.amenities_filename'), 'r');
+
+        if ($file) {
+            while (($line = fgets($file)) !== false) {
+                $values = explode(';', $line);
+                $buildingName = $values[0];
+                $amenityName = $values[1];
+
+                $amenity = Amenity::firstOrCreate([
+                    'name' => $amenityName
+                ]);
+                $building = Building::where([
+                    'name' => $buildingName
+                ])->firstOrFail();
+
+                $building->amenities()->firstOrCreate([
+                    'id' => $amenity->id
+                ]);
+            }
+
+            fclose($file);
+        } else {
+            die(config('scraper.command.amenities_filename') . ' file was not found');
+        }
+    }
+
+    /**
+     * @throws ModelNotFoundException
+     */
+    private function populateBuildingsImages(string $path): void
+    {
+        $file = fopen($path . config('scraper.command.images_filename'), 'r');
+
+        if ($file) {
+            while (($line = fgets($file)) !== false) {
+                try {
+                    $values = explode(';', $line);
+                    $hotelName = $values[0];
+                    $imageUrl = $values[1];
+
+                    $image = BuildingImage::firstOrNew([
+                        'url' => $imageUrl
+                    ]);
+                    $building = Building::where([
+                        'name' => $hotelName
+                    ])->firstOrFail();
+
+                    $building->images()->save($image);
+                } catch (Exception) {
+                    echo 'Could not find the building record associated with image' . "\n";
+                }
+            }
+            fclose($file);
+        } else {
+            die(config('scraper.command.images_filename') . ' file was not found');
```

Comment:
Doing `die($path. ' was not found')` would let the user know what went wrong much better.